documentation/usage.md

```diff
@@ -0,0 +1,73 @@
+<img src="https://user-images.githubusercontent.com/8014761/190516106-6e8c948c-9302-47e0-b09e-114a5456301d.png" alt="tourguide logo" style="width: 50%;">
+
+# Using TourGuide
+
+This file is meant to be a more detailed introduction to what TourGuide is and what kind of features TourGuide has. 
+
+## Walking Through TourGuide
+The easiest way to explain what TourGuide does is to show screenshots. To start, this is a view of what TourGuide looks like on initial run, for a user who is midway through a run.
+
+![image](https://user-images.githubusercontent.com/8014761/192012161-1a120f69-e811-47ac-9915-28c91ceb4cb2.png)
+
+At the top, TourGuide will tell you how many turns you've played in your ascension -- this helps speedrunners tell how they're doing on broader turncount relative to their overarching turncount goal. Underneath, it will start out by telling you your **Tasks**, which is the language TourGuide uses to represent things the user needs to do, most prominently main game quests -- as seen in the screenshot, things like the Boss Bat Quest, Knob Goblin Quest, Highland Lord Quest, etc.
+
+Each quest is associated with a tile. The tile changes as the user proceeds on the quest, giving different advice for different stages in the respective quest. Note that the boss bat tile is telling the user that there are just two extra zones that need to be unlocked -- the user has unlocked one chamber, but still needs sonars or screambats to unlock the Beanbat Chamber and the Boss Bat Chamber. If you were to click on the task tile, TourGuide will try to send you to the container zone where the quest is set -- for instance, if we were to click on Castle quest, we'd get this in our main window:
```

Comment:
This description of the boss bat quest does not seem to match up with the image provided (image shows 1 turn of delay left before boss can show up)

Reply:
fixed in latest commit, thank you. i was referencing the wrong screenshot lol x_x